myuw/dao/admin.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from myuw.util.settings import get_myuw_astra_group_stem,\
+    get_myuw_admin_group, get_myuw_override_group
+from myuw.dao import get_netid_of_original_user
+from myuw.dao.gws import gws
+
+
+logger = logging.getLogger(__name__)
+MYUW_ADMIN_GROUP = get_myuw_admin_group()
+MYUW_OVERRIDE_GROUP = get_myuw_override_group()
+MYUW_ASTRA_GROUP_STEM = get_myuw_astra_group_stem()
+
+
+def _search_groups(uwnetid):
+    group_names = set([])
+    group_refs = gws.search_groups(member=uwnetid,
+                                   stem=MYUW_ASTRA_GROUP_STEM,
+                                   scope="all",
+                                   type="effective")
+    is_admin = False
+    override = False
+    if group_refs:
+        for gr in group_refs:
+
+            if gr.name == MYUW_ADMIN_GROUP:
+                is_admin = True
+                break
```

Comment:
Could we change this to:

if gr.name == MYUW_ADMIN_GROUP:  
        override = true
        is_admin = True
        break

I would prefer it if the return values are not changed depending upon the order of the group_refs object in case another engineer uses this at some point in the future for a use case outside of its present one.

Reply:
If we're concerned about performance here, we could add an eq operator to the GroupRefence object and use the in keyword, right?

Reply:
Didn't see this early. Thanks for the suggestion! Changed.